web_src/js/features/notification.js

```diff
@@ -0,0 +1,67 @@
+const {AppSubUrl, csrf} = window.config;
+
+export async function initNotificationsTable() {
+  $('#notification_table .button').click(function () {
+    updateNotification(
+      $(this).data('url'),
+      $(this).data('status'),
+      $(this).data('page'),
+      $(this).data('q'),
+      $(this).data('notification-id')
+    ).then((data) => {
+      $('#notification_div').replaceWith(data);
+      initNotificationsTable();
+      updateNotificationCount();
+    });
+    return false;
+  });
+}
+
+export async function initNotificationCount() {
+  setInterval(() => {
+    updateNotificationCount();
+  }, 5000);
```

Comment:
It would be nice if this can be configured via api.ini

Reply:
Agreed - I think we need to have a think about what is an appropriate time here. I suspect that 5s is going to be too frequent.

Reply:
You can add server variables [here](https://github.com/go-gitea/gitea/blob/fcc8cdd446d99a822d835c6d36e8abc4b5ac42a6/templates/base/head.tmpl#L86-L97) to expose them to JS via `window.config.varname`.

Reply:
OK I've tried a different thing: start with 10s - if unchanged check again in 20s, 30s, 40s, 50s, and there on every 60s.